app/Http/Controllers/API/NodeController.php

```diff
@@ -226,6 +226,34 @@ public function allocations(Request $request)
          }
          return $allocations;
      }
+	 
+    /**
+     * List Node Allocation based on assigned to ID
+     *
+     * Returns a listing of the allocation for the specified server id.
+     *
+     * @Get("/nodes/allocations/{id}")
+     * @Versions({"v1"})
+     * @Response(200)
+     */
+     public function allocationsView(Request $request, $id)
+     {
+        $query = Models\Allocation::where('assigned_to', $id);
+        try {
+			
+			$allocations = $query->get();
+
+            if (!$allocations->first()) {
```

Comment:
Should be able to do `if(empty($query))` to accomplish the same thing, or even `if($query->count() < 1)`